components.html

```diff
@@ -0,0 +1,1094 @@
+              ><ion-icon name="download-outline"></ion-icon> Download</a
+            >
+          </li> -->
+        </ul>
+      </header>
+      <div class="container">
+        <aside class="sidebar">
+          <ul class="list-style">
+            <li>
+              <a
+                style="color: var(--primary-color)"
+                href="#introduction-section"
+                class="link-style"
+                id=""
+                ><h4>Introduction</h4></a
+              >
+            </li>
+            <li><h4>Components</h4></li>
+
+            <li class="category">
+              <a href="#avatar-section" class="link-style" id="avatar"
+                >Avatar</a
+              >
+            </li>
+            <li class="category">
+              <a href="#alert-section" class="link-style" id="alert">Alert</a>
+            </li>
+            <li class="category">
+              <a href="#badges-section" class="link-style" id="badge">Badge</a>
+            </li>
+            <li class="category">
+              <a href="#button-section" class="link-style" id="button"
+                >Button</a
+              >
+            </li>
+            <li class="category">
+              <a href="#card-section" class="link-style" id="card">Card</a>
+            </li>
+            <li class="category">
+              <a href="#image-section" class="link-style" id="image">Image</a>
+            </li>
+            <li class="category">
+              <a href="#input-section" class="link-style" id="input">Input</a>
+            </li>
+            <li class="category">
+              <a href="#list-section" class="link-style" id="lists">Lists</a>
+            </li>
+            <li class="category">
+              <a href="#nav-section" class="link-style" id="navigation"
+                >Navigation</a
+              >
+            </li>
+            <li class="category">
+              <a href="#modal-section" class="link-style" id="modal">Modal</a>
+            </li>
+            <li class="category">
+              <a href="#rating-section" class="link-style" id="rating"
+                >Rating</a
+              >
+            </li>
+            <!-- <li class="category">
+              <a href="#" class="link-style" id="grid">Grid</a>
+            </li> -->
+            <li class="category">
+              <a href="#text-section" class="link-style" id="uitilities"
+                >Text Utilities</a
+              >
+            </li>
+            <li class="category">
+              <a href="#snackbar-section" class="link-style" id="snackbar"
+                >Snackbar</a
+              >
+            </li>
+          </ul>
+        </aside>
+        <main class="main">
+          <section id="introduction-section">
+            <h2>Introduction</h2>
+            <div class="examples">
+              <h3>Html Boiler plate to get started:</h3>
+              <div class="code-snippet">
+                <iframe
```

Comment:
Can the styling on the code-snippet iframe be refactored out and placed in the css file?

Having all the style in one place will help later when you try to customize it further. Otherwise you might have to keep moving back and forth between your html and css. Some of this is repeating as well.